Library/Homebrew/cmd/prune.rb

```diff
@@ -1,5 +1,6 @@
 require "keg"
 require "cmd/tap"
+require "cmd/unlinkapps"
```

Comment:
You are right and I think it makes more sense to address this here in this PR (big chunks have been rewritten anyway) instead of deferring the code move to a new PR. I'll reshuffle things soon. Thanks for the soft nudge. :heart_eyes_cat: